### PR TITLE
Restrict max_group_size to be less than size in Action Selectors.

### PR DESCRIPTION
**Pull request by jonathan-dilorenzo**
See issue: #355

### CODE REVIEW

File: docs/v1/P4Runtime-Spec.mdk [hunk 1]
```diff
@@ -1612,9 +1612,10 @@ The `ActionProfile` message includes the following fields:
 
 * `max_group_size`, an `int32` representing the maximum sum of all member
   weights within any given selector group, in the case of an Action Selector, or
-  0 for an Action Profile. PSA programs can use the `@max_group_size` annotation
-  to provide this value for Action Selectors. If the annotation is omitted, the
-  P4Info field will default to 0.
+  0 for an Action Profile. `max_group_size` is never larger than `size`.
```
Comment (smolkaj) [inline docs/v1/P4Runtime-Spec.mdk:1615]: Can we phrase things a bit stronger: "is never larger" -> "must be no larger"?

Comment (smolkaj) [inline docs/v1/P4Runtime-Spec.mdk:1615]: This is orthogonal to your change, but while we're at it, can we discuss Action Profiles and Action Selectors in the same order as in the previous paragraph? Changing the ordering here is quite confusing

**Review by smolkaj — DISMISSED** [review]

File: docs/v1/P4Runtime-Spec.mdk (hunk 1 above)
Comment (jonathan-dilorenzo) [reply docs/v1/P4Runtime-Spec.mdk:1615]: Sure. Can do.

Comment (jonathan-dilorenzo) [reply docs/v1/P4Runtime-Spec.mdk:1615]: The main reason I went with this phrasing is that I mostly saw the imperative style of 'this is a fact' rather than 'make sure this is true' throughout the rest of the section. Though I now see several examples with "must", so sure!

**Review by antoninbas — APPROVED** [review]
LGTM, @smolkaj any follow-up comment?

**Comment by antoninbas** [comment]
@jonathan-dilorenzo I see that the CLA check is failing, you may want to get yourself added to the ONF - Google corporate CLA. Since I know you are a Google employee I can still merge this after I hear back from @smolkaj.

**Comment by jonathan-dilorenzo** [comment]
Sounds good. I will do so. Thanks!

**Comment by smolkaj** [comment]
Thanks @jonathan-dilorenzo, this looks great to me.
